gatsby-node.js

```diff
@@ -11,10 +11,17 @@ const path = require(`path`)
 const crypto = require('crypto')
 const remark = require('remark')
 const html = require('remark-html')
+const { paginate } = require('gatsby-awesome-pagination')
+const { postsPerPage } = require('./src/constants/blog.ts')
 
 const markdownProcessor = remark().use(html)
```

Comment:
This should come after all imports. there should be no newlines between all relative imports and can be a new line between package imports and relative imports